process/factory/shard/vmContainerFactory.go

```diff
@@ -172,16 +176,13 @@ func (vmf *vmContainerFactory) createInProcessArwenVMByEpoch(epoch uint32) (vmco
 }
 
 func (vmf *vmContainerFactory) replaceInProcessArwen(epoch uint32) {
-	for _, container := range vmf.containers {
-		newArwenVM, err := vmf.createInProcessArwenVMByEpoch(epoch)
-		if err != nil {
-			logVMContainerFactory.Error("cannot replace Arwen VM in epoch", "epoch", epoch)
-			continue
-		}
-
-		container.Replace(factory.ArwenVirtualMachine, newArwenVM)
+	newArwenVM, err := vmf.createInProcessArwenVMByEpoch(epoch)
```

Comment:
This implemented way to change the arwen binary will work only once. When the the epoch is the provided ArwenGasManagementRewriteEnableEpoch. Impossible to do a full import-db process without poking in the toml files. Should implement @sasurobert for a component that will decide that is time to replace the arwen binary even when the node does a rollback in a previous epoch.

Reply:
A new mechanism will be implemented to resolve the correct Arwen version for any epoch, similar to how `GasScheduleNotifier` works.